src/reactapp/src/paymentMethods/customRenderers.js

```diff
@@ -1,21 +1 @@
-import { paymentFetcher } from './utility';
-
-/**
- * Preparing custom renderers from the available payment methods configured
- * into this directory.
- *
- * Available payment methods are available in `paymentConfig.json`.
- *
- * Each payment methods will posses `renderers.js` file which will defines the
- * custom payment method renderers. This will collect all the renderers provide
- * it into the PaymentMethods component.
- */
-export default async function getCustomRenderers() {
-  const paymentMethodRenderers =
-    await paymentFetcher.fetchDataFromPaymentMethods('renderers.js', []);
-
-  return paymentMethodRenderers.reduce(
-    (renderers, item) => ({ ...renderers, ...item }),
-    {}
-  );
-}
+export default {};
```

Comment:
@rajeev-k-tomy Maybe add a note "Contents of this file will be overwritten by  `src/reactapp/scripts/configurePaymentMethods.js` automatically".

(and add that as a note to the new content when it's generated maybe? "Do not edit this file directly, it's auto generated by `src/reactapp/scripts/configurePaymentMethods.js`")

Reply:
Really good suggestion. This can be incorporated.

Reply:
Updated as per your suggestion

Reply:
Let me know the comment is fine.

Reply:
Almost perfect :D

`unlikely to be changed` > I understand you're saying it's unlikely you need to change this file, but it reads as "it is not likely this file will ever change", but it is in fact the opposite. The file will change *very likely* :)

Reply:
Applied the changes.